index.html

```diff
@@ -1,122 +1,901 @@
+		rel="icon" type="image/png" />
+	<style>
+		body {
+			font-family: 'Roboto', sans-serif;
+			background: url('/assets/cloud_bg.svg');
+			background-size: cover;
+			background-position: center;
+			background-attachment: fixed;
+			line-height: 1.6;
+		}
+
+		h1 {
+			font-size: 24px;
+			font-weight: bold;
+			margin-bottom: 0px;
+		}
+
+		h2 {
+			font-size: 18px;
+			font-weight: normal;
+			margin-top: 4px;
+
+		}
+
+		h3 {
+			font-size: 20px;
+			margin-bottom: 0px;
+		}
+
+		.next {
+			font-size: 15px;
+			margin-top: 8px;
+			margin-bottom: 8px;
+		}
+
+		.container {
+			max-width: 700px;
+			margin-left: auto;
+			margin-right: auto;
+		}
+
+		.hero {
+			margin-top: 100px;
+		}
+
+		.hero .logo {
+			width: 140px;
+			float: left;
+			margin-right: 32px;
+		}
+
+		.hero .message {
+			font-size: 15px;
+			text-align: center;
+			margin-bottom: 24px;
+		}
+
+		.details {
+			clear: both;
+			height: 400px;
+			line-height: 1.8;
+		}
+
+		.callout {
+			font-size: 14px;
+			background-color: #E8F0FE;
+			width: 100%;
+			color: #1967D2;
+			font-weight: 600;
+			padding: 16px 24px;
+			line-height: 1.6;
+		}
+
+		.callout label {
+			display: block;
+			padding-left: 2em;
+			text-indent: -2em;
+		}
+
+		.code {
+			font-family: 'Roboto Mono', Courier, monospace;
+			color: #F90056;
+			background-color: #F8F8F8;
+			border: 1px solid #DDD;
+			border-radius: 2px;
+			padding: 0 6px;
+			font-weight: 500;
+		}
+
+		.warning {
+			color: rgb(213, 0, 0);
+			background: #fbe9e7;
+			margin-bottom: 1.0em;
+			padding: 1.0em;
+		}
+
+		.warning a:visited {
+			color: rgb(213, 0, 0);
+		}
+
+		.warning a:link {
+			color: rgb(213, 0, 0);
+		}
+
+		.cta {
+			color: #3367D6;
+			text-align: left;
+			position: absolute;
+		}
+
+		.cta a {
+			color: white;
+			text-decoration: none;
+			background-color: rgb(66, 133, 244);
+			padding: 8px 12px;
+			margin-top: 8px;
+			min-height: 36px;
+			border-radius: 2px;
+			font-weight: 600;
+		}
+
+		.cta a:hover {
+			background-color: rgb(66, 133, 244, 0.9);
+		}
+
+		@media (max-width: 600px) {
+			.hero .logo {
+				float: initial;
+				display: block;
+				margin-left: auto;
+				margin-right: auto;
+			}
+
+			.container {
+				margin-top: 0;
+			}
+
+			.hero {
+				margin-top: 0;
+				margin-bottom: 24px;
+			}
+		}
+
+		.stargroup {
```

Comment:
Since this applies to the svg image itself, could you move the animation inside the svg file? 
See example at https://labs.steren.fr/2019/11/30/animating-svg-using-css/

Reply:
Moved animation to svg file, thanks for the pointer.